app/javascript/components/Calendar/index.js

```diff
@@ -145,11 +148,18 @@ const Calendar = ({
         events={selectEvents(events, currentUser, filters)}
         eventPropGetter={eventPropGetter}
         views={['month']}
-        components={calendarComponents(currentUser, offices, filters, {
-          changeShowFilter,
-          changeEventFilter,
-          changeOfficeFilter,
-        })}
+        culture="en"
```

Comment:
is this the culture default value or is it hard coded?

Reply:
we're just explicitly setting the culture here so in the future we can change it via an interface / menu. But I think "en" is the default